src/plugins/discover/public/application/components/download_csv/download_csv_popover_content.stories.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import type { ComponentStory } from '@storybook/react';
+import { DiscoverDownloadCsvPopoverContent } from './download_csv_popover_content';
+
+export default {
```

Comment:
are we sure the storybook stuff should be side by side to the source file. are we sure we are not bundling up the storybook stuff on release builds? could we verify that. just want to make sure we don't inflate the release.

Reply:
I have added a webpack config to exclude stories. great callout